scripts/tasks/src/eslint.ts

```diff
@@ -20,3 +21,15 @@ export const eslint = eslintTask({
   // (to display more, edit the `slice` call in the `display` function of node_modules/eslint/lib/linter/timing.js)
   timing: process.argv.includes('--timing'),
 });
+
+function hasProjectStories(root: string) {
+  const gitKeep = path.join(root, '.gitkeep');
+  if (!fs.existsSync(root)) {
+    return false;
+  }
+  if (fs.existsSync(gitKeep)) {
```

Comment:
not the most robust thing on the world. but I dont wanna glob parse folder tree on every lint run. this will be completely reworked in the future in a semantic fashin